### ISSUE TITLE
Recommended rules violations are not reported with webpack and eslint-loader

### ISSUE THREAD

**Issue by GaryB432** [typescript-eslint/typescript-eslint#657]
**Repro**

> webpack.config.js snippet
```JS
  module: {
    rules: [
      {
        test: /\.ts$/,
        loader: 'ts-loader',
      },
      {
        enforce: 'pre',
        test: /\.ts$/,
        exclude: /node_modules/,
        loader: 'eslint-loader',
        options: {
          formatter: FriendlyFormatter,
          parser: '@typescript-eslint/parser',
          extends: [
            'plugin:@typescript-eslint/recommended',
          ],
          plugins: ['@typescript-eslint'],
          // rules: { // works with rules like this
          //   'sort-keys': 2,
          //   '@typescript-eslint/explicit-function-return-type': 2,
          // },
          env: {
            browser: true,
            node: true,
          },
        },
      },
    ],
  },
```

```TS
const someObj = {
  b: 9,
  a: 4,  // member out of order
};

export function someNumber() { // no return type
  return undefined;
}

console.log(someNumber()!); // non-null assertion
```

**Expected Result**

All the violations of [recommended](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin) rules would be reported

**Actual Result**

No violations are reported

**Additional Info**

The violations are reported if I spell out the rules in the webpack config's `options.rules`.

Repro repo is available on [github](https://github.com/GaryB432/webpack-typescript-eslint.git)

> run npm build

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `1.11.0` |
| `@typescript-eslint/parser`        | `1.11.0` |
| `TypeScript`                       | `3.5.2` |
| `ESLint`                           | `6.0.1` |
| `node`                             | `10.16.0` |
| `npm`                              | `6.0.0` |
| `ts-loader` | `6.0.4`|
|`eslint-loader`|`2.1.2`|

**Comment by GaryB432** [comment]
an alternative title might be `webpack config options.extends seems to be not working`

**Comment by bradzacher** [comment]
This seems like an issue that belongs in the `eslint-loader` repo?

**Comment by GaryB432** [comment]
looks like they have an issue.  webpack-contrib/eslint-loader#272

I guess I'll hold off on `eslint@6` for now.  Oh well.

I am updating my yeoman generator from `tslint` and wanted to be up-to-date GaryB432/generator-webpack-ts

**Comment by bradzacher** [comment]
Ahh so it's a KP in the loader. Good to know!

I'll close this issue then.
Thanks for bringing this to our attention.

**Comment by GaryB432** [comment]
a KP?

**Comment by bradzacher** [comment]
Sorry, KP = Known Problem
